frontend/src/components/input-tag/index.tsx

```diff
@@ -50,86 +29,24 @@ export const TagLine: React.SFC<TagLineProps> = ({
     },
   );
 
-  if (loading) {
+  if (loading || error) {
```

Comment:
we should probably throw the error :)

Reply:
ah yes, I wanted to change it but temporarily used `|| error`

Reply:
done!